### DISCUSSION TITLE
Quick Navigation to "previous period" for all ranges

### DISCUSSION THREAD

**Discussion by TkDodo** [plausible/analytics#1323] [Anything Else]
I can see that currently, some ranges allow for a left/right (and arrow key) navigation. For example, it seems to work fine on the `today` view:

<img width="424" alt="Screenshot 2021-09-11 at 20 14 05" src="https://user-images.githubusercontent.com/1021430/132957377-232f268a-b91f-426c-a924-14e27a651883.png">

If I'm on `Today`, I can go to `Yesterday` and so on. The same is possible for the month view:

<img width="396" alt="Screenshot 2021-09-11 at 20 14 44" src="https://user-images.githubusercontent.com/1021430/132957390-1759bad2-4846-4253-b41d-5aa78bdea9eb.png">

but not for anything else, e.g. for the `last 7 days` view:

<img width="327" alt="Screenshot 2021-09-11 at 20 13 56" src="https://user-images.githubusercontent.com/1021430/132957406-3d04c6ed-4e3c-4823-9eb3-abee0e9f9886.png">

where a navigation to the 7 days before that etc. would be neat.

Ideally, it would work on all ranges, even custom ones, even though I admit finding out the intent of the range is not always easy. For example, selecting Monday-Friday:

<img width="309" alt="Screenshot 2021-09-11 at 20 14 22" src="https://user-images.githubusercontent.com/1021430/132957426-904f6489-c0b9-4f21-bba6-ed2ad3c4ca1a.png">

and then navigating to the left could mean that I'd like to see the Monday-Friday of the previous week, or that I'd like to see the 5 days prior to that (1st - 5th in the above example)

**Comment by ukutaht** [comment]
Yes this would be great.

> even though I admit finding out the intent of the range is not always easy.

I think we need explicit comparison modes like `Previous period`, `Month-over-month`, `Week-over-week`  etc. This would set what percentages we show as a comparison and also where the arrow will take you. What do you think?

**Reply by TkDodo** [reply]
I agree, this sounds like a good approach 👍
